customer-managed/aws/terraform/network.tf

```diff
@@ -61,7 +61,7 @@ resource "aws_subnet" "public" {
 }
 
 resource "aws_subnet" "private" {
-  count                   = length(var.private_subnet_cidrs)
+  count                   = length(var.private_subnet_ids) > 0 ? 0 : length(var.private_subnet_cidrs)
```

Comment:
I'm not following this logic. If the length is greater than 0, you're setting zero, otherwize it's the length (which would be zero?)

Reply:
Oh, wait, sorry, I misread the var.  Ignore me, lol

Reply:
possible i have it tangled up :) I still need to test it

Reply:
If the customer passes pre-existing subnets, in the form of ids... do not create any subnets. Else create one per cidr that was passed.